src/Shared/Shared.csproj

```diff
@@ -42,4 +43,8 @@
     <PackageReference Update="Nerdbank.GitVersioning" Version="3.1.91" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Properties\" />
```

Comment:
Remove this? I don't see any files you added in this folder.

Reply:
Will remove...VS is very persistent adding it back :S